RELEASE.rst

```diff
@@ -1,6 +1,24 @@
 Release Notes
 -------------
 
+Version 0.3.0
+=============
+
+- Added UI to add and remove repository members.
+- Added form for adding new vocabularies.
+- Added manage taxonomies panel and button
+- Tests for members REST.
+- REST for repo members
+- Implemented taxonomy model delete cascading.
+- Renamed &#39;Copy to Clipboard&#39; to &#39;Select Text&#39;
```

Comment:
Also, as @pwilkins pointed out, the new button is "Select XML" not "Select Text"